packages/rxjs/spec/operators/share-spec.ts

```diff
@@ -810,9 +797,22 @@ describe('share', () => {
       });
     });
 
-    it('should not reset on refCount 0 if reset notifier errors before emitting any value', () => {
-      spyOnUnhandledError((onUnhandledError) => {
+    describe('when config.onUnhandledError is set', () => {
```

Comment:
Adding this `describe` block is what I think messed with the whitespace diff in other parts of the file.